### DISCUSSION TITLE
questions on formatting

### DISCUSSION THREAD

**Discussion by hobermad** [Obsidian-TTRPG-Community/fantasy-statblocks#209] [Q&A] [answered]
I'm new to obsidian, and found your plugin. I'm pretty excited about using it for Fate, and experimenting with creating a Fate Condensed Layout. I have two questions:

The first is where do you put the parameter columns, to set for 2? Your fate core example displays in two columns in blue and grey, but I copied the example file, and it displays only in 1 column and in the red text on beige. Can you provide an example of where I should define 'columns: 2" and how can I use the same color scheme as your fate core layout?
<img width="325" alt="Screenshot 2023-04-30 003353" src="https://user-images.githubusercontent.com/25756724/235335672-ee69937c-88dc-403b-a31f-625d0ef0051d.png">

**Comment by sigrunixia** [comment, marked as answer]
Hello!

To get multiple columns, you can use the following within your code block.

```yaml
columns: 2
forceColumns: true
```

More documentation on columns (and Statblocks in [general here](https://plugins.javalent.com/statblocks) Our readme will be overhauled soon.).

As far as the colors, it does appear the dark colors are not showing up right at the moment. I will check into that and see what we can do to get that fixed.

**Reply by sigrunixia** [reply]
An issue has been created for the coloration. I'll try to get it fixed as bugs allow.
